src/App.jsx

```diff
@@ -30,11 +30,10 @@ function App() {
   }, [isDarkMode]);
 
   return (
-    <div className={`d-flex flex-column ${isDarkMode ? 'dark' : ''}`}>
-      <Router>
+    <div id="parentLayout" className={`d-flex flex-column ${isDarkMode ? 'dark' : ''}`}>
+      <LoadingBar height={3} color="#f11946" progress={progress} />
+      <Router> 
         <Navbar />
-        <LoadingBar height={3} color="#f11946" progress={progress} />
```

Comment:
?

Reply:
As you can see the webpages are now in a proper layout.
Making a new PR - with the requested doubt - reverted back to the previous line only the LoadingBar
Else, the footer will stay at the bottom. - as added layout.

Jump to 8 seconds for video. 

https://github.com/user-attachments/assets/96295107-4dc7-43b5-83c8-bfe182ade3b3

Either way - the layout is good now.